src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/EventMessageReceiverServiceTest.java

```diff
@@ -438,48 +414,28 @@ void should_handle_dlq_case_event_asb_message_when_feature_flag_enabled_and_inva
     @Test
     void should_handle_dlq_message_when_error_parsing_user_id()
             throws JsonProcessingException {
-        when(featureFlagProvider.getBooleanValue(DLQ_DB_INSERT, null))
-                .thenCallRealMethod();
 
         final NullPointerException nullPointerException = assertThrows(NullPointerException.class, () ->
                 eventMessageReceiverService.handleDlqMessage(MESSAGE_ID, SESSION_ID, MESSAGE_WITHOUT_USER));
 
-        assertEquals("userId is null", nullPointerException.getMessage());
+        assertEquals(null, nullPointerException.getMessage());
         verifyNoInteractions(caseEventMessageRepository);
     }
 
     @Test
     void should_handle_dlq_message_when_missing_user_id() {
 
-        when(featureFlagProvider.getBooleanValue(DLQ_DB_INSERT, null))
-                .thenCallRealMethod();
-
         final NullPointerException nullPointerException = assertThrows(NullPointerException.class, () ->
                 eventMessageReceiverService.handleDlqMessage(MESSAGE_ID, SESSION_ID, MESSAGE_WITHOUT_USER));
 
-        assertEquals("userId is null", nullPointerException.getMessage());
+        assertEquals(null, nullPointerException.getMessage());
         verifyNoInteractions(caseEventMessageRepository);
     }
 
-    @Test
-    void should_handle_ccd_case_event_asb_message_when_feature_flag_disabled() {
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.

Reply:
We have removed the logic of R1 route code as part of this changes, so R1 tests (when_feature_flag_disabled) have to be removed too.